themes/src/main/resources/theme/base/login/user-profile-commons.ftl

```diff
@@ -0,0 +1,22 @@
+<#macro userProfileField attribute>
+<div class="${properties.kcFormGroupClass!}">
```

Comment:
Why not also include here the `list` directive. Shouldn't it be also part of this directive?

Reply:
I did it this way because of password field in `register.ftl`, but you are probably right, I can define some "callbacks" using `<#nested "aaa">` directive so page templates can extend this loop. Going to try this approach.

Reply:
updated